contracts/simpleKitty/SimpleKittyBase.sol

```diff
@@ -52,16 +52,15 @@ contract SimpleKittyBase {
         // numbers of their parents, plus one.
         // (i.e. max(matron.generation, sire.generation) + 1)
         uint16 generation;
+        bool isGestating;
```

Comment:
it is not necessary to store `isGestating` - this is a calculated value: `kit.siringWithId != 0`

Reply:
Fixed https://github.com/poanetwork/cryptokitties-xdai-demo/pull/5/commits/77c8b93adff0157764860d6569e92c96b6750275